app.yml

```diff
@@ -135,4 +135,4 @@ url: https://github.com/github/safe-settings
 
 # Set to true when your GitHub App is available to the public or false when it is only accessible to the owner of the app.
 # Default: true
-public: false
```

Comment:
Could we keep this as private? i.e. keep the default posture as app being private